css/styles.css

```diff
@@ -0,0 +1,61 @@
+.container-fluid {
+    background-color: yellow;
+}
+
+header {
+    background-color: blue;
+}
+
+nav {
+    background-color: green;
+}
+
+section .col-md-8 {
+    background-color: red;
+}
+
+aside {
+    background-color: aqua;
+}
+
+.promo {
+    background-color: orange;
+}
+
+footer {
+    background-color: black;
+    color: white;
+}
+
+header,
+.col-md-3 {
+    height: 150px;
+}
+
+nav,
+footer {
+    height: 50px;
+}
+
+section .col-md-8,
+aside {
+    height: 200px;
+}
+
+/* header,
+nav,
+.col-md-3,
+aside,
+section,
+footer {
+    border: 2px solid #000;
+} */
+
+.black-border, .promo {
+    border: 2px solid #000;
+}
+Collapse
+
+
+
+Message Input
```

Comment:
This looks great but go ahead and delete the `Collapse` and `Message Input` as you see at the bottom here due to the fact that it can possibly break code since they're not real CSS syntax.